cmd/minikube/cmd/addons/disable.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package addons
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/docker/machine/libmachine"
+	"github.com/docker/machine/libmachine/drivers"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"k8s.io/minikube/cmd/minikube/cmd/config"
+	"k8s.io/minikube/pkg/minikube/assets"
+	"k8s.io/minikube/pkg/minikube/cluster"
+	"k8s.io/minikube/pkg/minikube/constants"
+	"k8s.io/minikube/pkg/minikube/sshutil"
+)
+
+var addonsDisableCmd = &cobra.Command{
+	Use:   "disable ADDON_NAME ADDON_VALUE",
+	Short: "Disables an individual value in a minikube addons file",
+	Long: `Disables the ADDON_NAME addons value to ADDON_VALUE
+	These values can be overwritten by flags or environment variables at runtime.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) != 1 {
+			fmt.Fprintln(os.Stderr, "usage: minikube addons disable ADDON_NAME")
+			os.Exit(1)
+		}
+
+		addon := args[0]
+		err := disable(addon)
+		if err != nil {
+			fmt.Fprintln(os.Stdout, err)
+		}
+		fmt.Fprintln(os.Stdout, fmt.Sprintf("%s was successfully disabled", addon))
+	},
+}
+
+func init() {
+	AddonsCmd.AddCommand(addonsDisableCmd)
+}
+
+func disable(name string) error {
+	if addon, ok := assets.Addons[name]; ok {
+		// check if already disabled? (not sure if better or worse....)
+		if err := config.Set(name, "false"); err != nil {
+			return err
```

Comment:
I think its fine to check if its already disabled.  Should this just be a noop return?